moveit_core/trajectory_processing/include/moveit/trajectory_processing/time_optimal_trajectory_generation.h

```diff
@@ -207,6 +207,9 @@ class TimeOptimalTrajectoryGeneration : public TimeParameterization
                                           const Eigen::VectorXd& max_velocity,
                                           const Eigen::VectorXd& max_acceleration) const;
 
+  /// @brief Check if a combination of revolute and prismatic joints is used. path_tolerance_ is not valid, if so.
+  bool checkMixedUnits(const moveit::core::JointModelGroup* group) const;
```

Comment:
The function name and comment does not really specify what to expect from the bool (it's the opposite of what I was expecting). Could you rename this to `hasMixedJointTypes()` and directly return `(have_prismatic && have_revolute)` from the function to streamline the logic? Obviously, this flips the bool from the current implementation, but it would improve redability.